[response_started_tentative.py]
  expected: TIMEOUT
